backend/protocol_rpc/endpoints.py

```
@@ -1732,245 +1407,172 @@ def get_contract(consensus_service: ConsensusService, contract_name: str) -> dic
+    )
+    register_rpc_endpoint(
+        partial(get_balance, accounts_manager),
+        method_name="eth_getBalance",
+    )
+    register_rpc_endpoint(
+        partial(get_transaction_by_hash, transactions_processor),
+        method_name="eth_getTransactionByHash",
+    )
+    register_rpc_endpoint(
+        partial(
+            call, request_session, accounts_manager, msg_handler, transactions_parser
```

Ruff (0.15.2)

[error] Undefined name `call`

(F821)